frontend/components/Domain/User/UserInviteDialog.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+  <BaseDialog
+    v-model="inviteDialog"
+    :title="$tc('profile.get-invite-link')"
+    :icon="$globals.icons.accountPlusOutline" color="primary">
+    <v-container>
+      <v-form class="mt-5">
+        <v-select
+v-if="groups && groups.length" v-model="selectedGroup" :items="groups" item-text="name"
+          item-value="id" :return-object="false" filled :label="$tc('group.user-group')"
+          :rules="[validators.required]" />
+        <v-select
+v-if="households && households.length" v-model="selectedHousehold" :items="filteredHouseholds"
+          item-text="name" item-value="id" :return-object="false" filled :label="$tc('household.user-household')"
+          :rules="[validators.required]" />
+        <v-row>
+          <v-col cols="9">
+            <v-text-field :label="$tc('profile.invite-link')" type="text" readonly filled :value="generatedSignupLink" />
+          </v-col>
+          <v-col cols="3" class="pl-1 mt-3">
+            <AppButtonCopy
+:icon="false" color="info" :copy-text="generatedSignupLink"
+              :disabled="generatedSignupLink" />
+          </v-col>
+        </v-row>
+        <v-text-field
+v-model="sendTo" :label="$t('user.email')" :rules="[validators.email]" outlined
+          @keydown.enter="sendInvite" />
+      </v-form>
+    </v-container>
+    <template #custom-card-action>
+      <BaseButton :disabled="!validEmail" :loading="loading" :icon="$globals.icons.email" @click="sendInvite">
+        {{ $t("group.invite") }}
+      </BaseButton>
+    </template>
+  </BaseDialog>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, useContext, ref, toRefs, reactive } from "@nuxtjs/composition-api";
+import { watchEffect } from "vue";
+import { useUserApi } from "@/composables/api";
+import BaseDialog from "~/components/global/BaseDialog.vue";
+import AppButtonCopy from "~/components/global/AppButtonCopy.vue";
+import BaseButton from "~/components/global/BaseButton.vue";
+import { validators } from "~/composables/use-validators";
+import { alert } from "~/composables/use-toast";
+import { GroupInDB } from "~/lib/api/types/user";
+import { HouseholdInDB } from "~/lib/api/types/household";
+import { useGroups } from "~/composables/use-groups";
+import { useAdminHouseholds } from "~/composables/use-households";
+
+export default defineComponent({
+  name: "UserInviteDialog",
+  components: {
+    BaseDialog,
+    AppButtonCopy,
+    BaseButton,
+  },
+  props: {
+    value: {
+      type: Boolean,
+      default: false,
+    },
+    isAdmin: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
You should be able to detect this instead of passing a prop. In setup:

```js
    const { $auth, i18n } = useContext();
    const isAdmin = computed(() => $auth.user?.admin);
```